common-requests/get-technical-support.md

```diff
@@ -1,8 +1,20 @@
 ---
 layout: page
 title: "Technical Support"
-description: "TBD"
+description: ""
+tags:
+ - Agent
+ - Help
+ - Support
+author: edeediong
 ---
 
-# Get Technical Support
+## Get Technical Support
 
+To get technical support on Open Horizon, join the LF Edge [Slack workspace](https://join.slack.com/t/lfedge/shared_invite/zt-7kavdtmq-SeyFzM2CEABBcKYGEVCgkw) if you haven't.
```

Comment:
Chg to:

For Open Horizon technical support, join the LF Edge [Slack workspace](https://join.slack.com/t/lfedge/shared_invite/zt-7kavdtmq-SeyFzM2CEABBcKYGEVCgkw).

Reply:
Done.